app/jobs/calendar_importer/events/base.rb

```diff
@@ -26,24 +26,24 @@ def sanitize_invalid_char(input)
     # Strip out all shady tags
     # Convert all html to markdown
     def html_sanitize(input)
-      return if input.blank?
+      return '' if input.blank?
 
-      allowed_tags = %w[a strong b em i ul ol li blockquote h3 h4 h5 h6 br]
+      clean_input = sanitize_invalid_char(input)
 
-      str = Nokogiri::HTML.fragment(input)
-      str.css(*['h1', 'h2']).each { |header| header.name = 'h3' }
+      doc = Nokogiri::HTML.fragment(clean_input)
+      doc.css('h1', 'h2').each { |header| header.name = 'h3' }
 
       if footer.present?
-        str << '<br/><br/>'
-        str << footer
+        doc << '<br/><br/>'
+        doc << footer
       end
 
-      str = str.to_s
+      body_text = doc.serialize
 
-      str = ActionController::Base.helpers.sanitize(str, tags: allowed_tags)
-      str = sanitize_invalid_char(str)
+      allowed_tags = %w[p a strong b em i ul ol li blockquote h3 h4 h5 h6 br]
```

Comment:
This feels like something we should have stored somewhere thats not hidden in a function. Maybe just move to a const?